### PR TITLE
Fix comments and function names are different

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fixed two minor bugs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func getKubeConfigSpecs(cfg *kubeadmapi.InitConfiguration) (map[string]*kubeConf
 		kubeadmconstants.KubeletKubeConfigFileName: {
 			CACert:     caCert,
 			APIServer:  controlPlaneEndpoint,
-			ClientName: fmt.Sprintf("%s%s", kubeadmconstants.NodesUserPrefix, cfg.NodeRegistration.Name),
+			ClientName: fmt.Sprintf("%s %s", kubeadmconstants.NodesUserPrefix, cfg.NodeRegistration.Name),
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go]: can you please explain your change here?
how is this a bug?

@kubernetes/sig-cluster-lifecycle-pr-reviews

**Review by neolit123 — COMMENTED** [review]
/hold

File: cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go (hunk 1 above)
Comment (xichengliudui) [reply cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go]: Maybe I was wrong , I think there should be a space between the two parameters.

Comment (xichengliudui) [reply cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go]: @neolit123  Shall I change it back?

Comment (fabriziopandini) [reply cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go]: This should not be changed. See https://kubernetes.io/docs/reference/access-authn-authz/node/

**Comment by xichengliudui** [comment]
Sorry, I have updated it.

**Comment by yagonobre** [comment]
@xichengliudui due the scope change, can you rewrite your commit message/pr title?
/priority backlog

**Comment by neolit123** [comment]
please change the commit message/title too and force push.
thanks.

**Comment by xichengliudui** [comment]
Has been updated.

**Review by neolit123 — COMMENTED** [review]
thanks
/lgtm
/approve
/hold cancel

**Comment by yagonobre** [comment]
@xichengliudui do you forget to push force?

**Comment by xichengliudui** [comment]
I'm a little confused, Why push?

**Comment by xichengliudui** [comment]
Now there is only one submission and I don't need to compress my submission, right?

**Comment by yagonobre** [comment]
To update your commit message

**Comment by xichengliudui** [comment]
ok.

**Comment by SataQiu** [comment]
Thanks @xichengliudui 
But you need to solve the conflict first.

**Comment by xichengliudui** [comment]
@SataQiu  LGTM

**Comment by yagonobre** [comment]
Thanks @xichengliudui, please squash your commits and we can get it merged.
/remove-area e2e-test-framework
/remove-area test
/remove-sig scheduling
/remove-sig testing

**Pull request closed without merging**